librz/core/cfile.c

```diff
@@ -1027,44 +1057,48 @@ RZ_API bool rz_core_bin_load(RzCore *r, const char *filenameuri, ut64 baddr) {
 	if (!rz_config_get_b(r->config, "cfg.debug")) {
 		loadGP(r);
 	}
-	if (rz_config_get_i(r->config, "bin.libs")) {
+	if (rz_config_get_b(r->config, "bin.libs")) {
 		const char *lib;
 		RzListIter *iter;
 		RzList *libs = rz_bin_get_libs(r->bin);
 		rz_list_foreach (libs, iter, lib) {
 			if (file_is_loaded(r, lib)) {
 				continue;
 			}
-			eprintf("[bin.libs] Opening %s\n", lib);
+			RZ_LOG_INFO("Opening library %s\n", lib);
 			ut64 baddr = rz_io_map_location(r->io, 0x200000);
 			if (baddr != UT64_MAX) {
 				rz_core_file_loadlib(r, lib, baddr);
 			}
 		}
-		rz_core_cmd0(r, "obb 0;s entry0");
-		rz_config_set_i(r->config, "bin.at", true);
-		eprintf("[bin.libs] Linking imports...\n");
+
+		rz_core_cmd0(r, "ob 0; s entry0");
```

Comment:
Please use the API

Reply:
same here. the whole thing needs to be rewritten. i would avoid changing this for now.